src/codegen/mod.rs

```diff
@@ -90,6 +90,8 @@ pub struct Options {
     pub opt_level: OptimizationLevel,
     pub log_api_return_codes: bool,
     pub log_runtime_errors: bool,
+    pub log_prints: bool,
+    pub release: bool,
```

Comment:
I'm not sure this is needed ATM? This is just set but then never read out again to change something in the code generation, so might as well get rid of it.

Reply:
Do you mean `log_prints` ? it is used to disable putting print instructions in the cfg. 
Check: https://github.com/salaheldinsoliman/solang/blob/bc705be1d56fb80758febc4d0a23a2e19faa29c9/src/codegen/expression.rs#L825